schemas/models/structured_model.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/bettyblocks/json-schema/master/schemas/models/structured_model.json",
+  "title": "Question",
+  "description": "A question in a questionnaire to be asked to a user",
```

Comment:
Vague description, we could improve on this

Reply:
Let me change it.

Reply:
Done.